src/leiningen/new/chestnut/dev/user.clj

```diff
@@ -1,5 +1,10 @@
 (ns user
   (:require [{{project-ns}}.server]
+            [com.stuartsierra.component :as component]
+            [figwheel-sidecar.config :as config]
+            [figwheel-sidecar.system :as sys]
```

Comment:
I think `sys` is confusing when reading the code, as one could think it refers to System. Maybe call these `fw-config` and `fw-sys`?